### ISSUE TITLE
Support multiple file uploads per request

### ISSUE THREAD

**Issue by atz** [projecthydra-labs/hyrax#365]
This is now a supported part of CarrierWave.  See: https://www.codementor.io/tips/8367241728/carrierwave-now-natively-supports-multiple-file-uploads-in-rails-gotchas

Would have to change the `files` field type to JSON.  Jquery FileUploader seems like it would already be capable of taking advantage of this.

**Comment by mjgiarlo** [comment]
Yes, the jquery-file-upload component can do this. I believe we can set this option to `false`: https://github.com/blueimp/jQuery-File-Upload/wiki/Options#singlefileuploads
